docs/source/tutorial/09-write-your-first-mutation.mdx

```diff
@@ -2,43 +2,58 @@
 
-This mutation takes a single argument, the `email` address of the person being logged in. Unlike many GraphQL operations that return objects which need to have fields selected, the `login` mutation returns only a single string.
+Click the play button to the right to open that mutation in the Explorer tab. When it opens, click the plus sign to add the operation: 
 
-Type the following mutation in GraphiQL:
+<img alt="The login mutation after initially being added" class="screenshot" src="images/explorer_added_login_mutation.png"/>
 
-```graphql
-mutation Login($email: String) {
-  login(email: $email)
-}
-```
+Sandbox Explorer tries to be helpful by adding the word `Mutation` after the word `Login`, but the Android SDK also does that for you - so you'd wind up with a `LoginMutationMutation`. Rename the operation `Login` instead: 
+
+<img alt="The renamed operation" class="screenshot" src="images/explorer_login_mutation_rename.png"/>
+
+Notice that there are no brackets after the call to the `login` mutation - this is because the type returned by this mutation is `String`, which is known as a **scalar type**. This means it won't have properties the way an object would, so you don't need to specify what properties you want returned to you. 
+
+You'll also notice that `email` wasn't automatically added as an argument even though it doesn't seem to have a default value: That's because `email` is of type `String` - which remember, in GraphQL, means that it's **not** required (although obviously you won't get too far without it).  
 
-If you hit Play, you should get a `null` login:
+Click the plus sign next to the `email` argument to have that argument added: 
 
-<img alt="Results of not passing email" class="screenshot" src="images/login_mutation_null.png"/>
+<img alt="The operation with the email argument" class="screenshot" src="images/explorer_login_email_added.png"/>
 
-This is expected, because you didn't specify your email. To do so, add it to the **Query Variables**  in the lower-left pane of GraphQL Playground:
+You'll also notice that Sandbox Explorer has added a variable to your "Variables" section to match the login email. 
 
-```json:title=(GraphiQL)
-{ "email": "me@example.com" }
+Click the Submit Operation button your mutation. You'll see that  since you sent `null` for the email address, you get back `null` for the login: 
+
+<img alt="Results of passing a null email" class="screenshot" src="images/login_mutation_null.png"/>
+
+Now, replace `null` in the Query Variables section with an actual email address:
+
+```json:title=(Sandbox%20Explorer)
+{ "loginEmail": "me@example.com" }
 ```
 
-Press the Play button, and you'll get an actual response:
+Press the Submit Operation button, and this time you'll get an actual response:
 
 <img alt="Results of passing an actual email" class="screenshot" src="images/login_mutation_email.png"/>
 
+Next, copy the operation, either manually or using the meatball menu's "Copy operation" option. 		
```

Comment:
Very nitpick: didn't know this expression 😅 - I would have used "..." but maybe that can be confusing too 😄

Reply:
ah yeah i reference the meatball menu elsewhere in the iOS tutorial, but I guess not here. I'll update